tfhe/c_api_tests/test_forward_compatibility.c

```diff
@@ -0,0 +1,120 @@
+#include <tfhe.h>
+
+#include <assert.h>
+#include <inttypes.h>
+#include <stdio.h>
+
+#ifdef DEFINE_FORWARD_COMPATIBILITY
```

Comment:
very minor thing: I would prefer `WITH_FEATURE_FORWARD_COMPATIBILITY`, the `#ifdef DEFINE` feels like a repetition to me

Reply:
ok will rename then, it was just the cbindgen style